src/Workbench/Canvas/RenderHandler.cpp

```diff
@@ -0,0 +1,93 @@
+#include "clay.h"
+#include "assert.h"
+
+#include "raylib.h"
+#include "rlgl.h"
+#include "raymath.h"
+#include "rcamera.h"
+
+#include <array>
+#include <vector>
+#include <string>
+#include <sstream>
+#include "fmt/format.h"
+
+#include "Scene.h"
+#include "CanvasHelpers.cpp"
+#include "RenderHandler.h"
+
+void RenderCanvas(Scene &scene)
+{ 
+    static Model example_model = LoadModel("../src/stand_dock_model.obj");
+    auto modelColor = (Color){140,140,140,255}; 
+    auto wireframeColor = (Color){140,140,140,100};
+
+    DrawModel(example_model, {0, 0, 0}, 1.0f, modelColor);
+    DrawModelWires(example_model, {0, 0, 0}, 1.0f, wireframeColor);
+    UI::DrawOriginPlane(UI::OriginPlane::XZ, {0,0,0}, {10, 10}, Color{20, 20, 100, 100});
+    UI::DrawGrid(UI::OriginPlane::XZ, 100, 1.0f);
+}
+
+void CanvasRenderHandler(Clay_RenderCommand * renderCommand)
```

Comment:
```suggestion
void CanvasRenderHandler(Clay_RenderCommand* renderCommand)
```

These should either all be with the type or the paramater's name. Once sonarqube is integrated these should be easier.